src/styles/getModalStyles.js

```diff
@@ -96,6 +96,24 @@ export default (type, windowDimensions) => {
             animationIn = 'fadeInLeft';
             animationOut = 'fadeOutLeft';
             break;
+        case CONST.MODAL.MODAL_TYPE.RIGHT_DOCKED:
+            modalStyle = {
+                margin: 0,
+                flexDirection: 'row-reverse',
```

Comment:
Is this doing anything? I think it would be `flex-end` that would move the modal to the right of the screen.

Reply:
I was following the flex-direction example here: https://reactnative.dev/docs/flexbox#flex-direction

If I remove this then the modal goes to the left side.

Reply:
Cool. I guess it works then? If your width was 100% of the screen then I'd think flex-end would stick it at the right side but didn't really dig too much into what this is doing

Reply:
I think I see what your saying. Switched this up to use `row` and `end` below.